README.md

```diff
@@ -54,5 +54,31 @@ will begin to drift slightly from the original design. This is expected and
 there is no expectation that the original design proposal needs to be updated
 to reflect these differences.
 
-The code and our documentation are the ultimate sources of truth. CFPs are merely
-the starting point for the implementation.
+The code and our documentation are the ultimate sources of truth. CFPs are
+the starting point for the implementation, but can also be updated as needed.
+
+# Status
+
+The status of a CFP indicates its maturity. There are four different statuses
+that a CFP can have:
+
+1. Draft: A CFP in any form (Google doc, Github issue, ect.) that has not yet
```

Comment:
See my comment above -- it would be good to mandate that a Github issue be opened to track progress from Draft state i.e. make it a MUST that there is an issue. This makes it very easy for the project to track all draft proposals that pending (e.g. for triage).

Reply:
Agree. I think we already have this expectation at some level, because the number of the CFP comes from the issue number on the corresponding repository. Even if it's a stub we should just make that part of the standard workflow.

Reply:
(Worth noting though that for any formal CFP there will be a PR in this repository, so tracking feels like it should be relatively straightforward by just listing the open PRs in this repository... but regardless of that I think this is worthwhile.)

Reply:
updated the wording slightly to make Github issue MUST